views/pages/add-user.php

```diff
@@ -0,0 +1,78 @@
+
+<?php
+
+require_once '../../config/dbauth.php';
+
+$conn = new mysqli($hn, $un, $pw, $db);
+if($conn->connect_error) die($conn->connect_error);
+
+
+
+function add_user($conn, $AccountID, $FirstName, $LastName, $Phone,  $Email, $Street, $City, $State, $Zip,  $StartDate, $token, $AccountTypeID){
+	//code to add user here
+	$query = "INSERT INTO LIB_ACCOUNT (AccountID, FirstName, LastName, Phone, Email, Street, City, State, Zip, StartDate, Password, AccountTypeID) VALUES ('$AccountID', '$FirstName', '$LastName', '$Phone', '$Email', '$Street', '$City', '$State', '$Zip', '$StartDate', '$token', '$AccountTypeID')";
+	$result = $conn->query($query);
+	if(!$result) die($conn->error);
+}
+
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+	$AccountID = $_POST['AccountID'];
+	$FirstName = $_POST['FirstName'];
+	$LastName = $_POST['LastName'];
+	$Phone = $_POST['Phone'];
+	$Email = $_POST['Email'];
+	$Street = $_POST['Street'];
+	$City = $_POST['City'];
+	$State = $_POST['State'];
+	$Zip = $_POST['Zip'];
+	$StartDate = $_POST['StartDate'];
+	$Password = $_POST['Password'];
+	$AccountTypeID = $_POST['AccountTypeID'];
+
+	
+	$token = password_hash($Password,PASSWORD_DEFAULT); 
+
+	add_user($conn, $AccountID, $FirstName, $LastName, $Phone,  $Email, $Street, $City, $State, $Zip,  $StartDate, $token, $AccountTypeID);
+	
+		
+
+	
+header("Location: home.php");
+exit();
+
+
+
+
+}
+?>
+
+<html>
+	<head>
+	
+	</head>
+	
+	<body>
+		<form method='post' action='add-user.php'>
+			<pre>
+				AccountID: <input type='text' name='AccountID'>
```

Comment:
The AccountID is determined by the database, no? Via the `AUTO_INCREMENT` constraint? Additionally, no user registering for an account should be able to determine their own AccountID